tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -4695,30 +4695,45 @@ void Fill(const RuntimeShape& value_shape, const T* value_data,
 }
 
 template <typename Scalar>
-void Reverse(int axis, const RuntimeShape& input_shape,
-             const Scalar* input_data, const RuntimeShape& output_shape,
-             Scalar* output_data) {
+void Reverse(std::vector<bool>& axes, int input_size,
+             const RuntimeShape& input_shape, const Scalar* input_data,
+             const RuntimeShape& output_shape, Scalar* output_data) {
   gemmlowp::ScopedProfilingLabel label("Reverse");
+  int axes_size = static_cast<int>(axes.size());
+  int axis = 0;
+
+  Scalar* input_copy = (Scalar*)malloc(input_size * sizeof(Scalar));
```

Comment:
I think this memory allocation slows down the existing "Reverse" (that supports only one axis). Maybe add a check to see if the extra memory is needed or not.

Also, I am wondering if this malloc in the kernel can be avoided at all.